.github/workflows/run-integration-bi.yml

```
@@ -0,0 +1,181 @@
+        run: |
+          pip install -r requirements.txt
+          pip install datus-postgresql
+          pip install datus-semantic-metricflow
+
+      - name: Setup config file
+        run: |
+          cp /home/datus_ci/ci_template/agent.yml conf/agent.yml
+          cp /home/datus_ci/ci_template/ci.env .env
+
+      - name: Load environment variables
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:9:21: Double quote to prevent globbing and word splitting

(shellcheck)